test/env.sh

```diff
@@ -13,11 +13,11 @@
 : "${VPC_PRIVATE_SUBNET_STRING:=}"
 : "${EKS_CLUSTER_K8_VERSION:=1.31}"
 # Below env variables required to run license master test cases
-: "${ENTERPRISE_LICENSE_S3_PATH:=}"
-: "${TEST_S3_BUCKET:=}"
+: "${ENTERPRISE_LICENSE_S3_PATH:=/test_licenses}"
+: "${TEST_S3_BUCKET:=splk-test-data-bucket}"
 # Below env variables required to run remote indexes test cases
-: "${INDEXES_S3_BUCKET:=}"
-: "${AWS_S3_REGION:=}"
+: "${INDEXES_S3_BUCKET:=splk-integration-test-bucket}"
```

Comment:
Are we hardcoding these for now? Not a bad idea

Reply:
yes